### PR TITLE
Fix bug 1233185 - Remove explanation text from search results.

### PR DESCRIPTION
**Pull request by jezdez**
This simplifies the rendering dramatically and fixes a regression in the DRF 3.x update.

### CODE REVIEW

File: kuma/search/templates/search/results.html [hunk 1]
```diff
@@ -34,7 +34,7 @@
         <h4><a href="{{ doc.url }}"{% if index == 1 %} tabindex="1"{% endif %}>{{ doc.title }}</a></h4>
         <p>{{ doc.excerpt|safe }}</p>
         <p class="search-meta"><a href="{{ doc.url }}">{{ doc.url|replace('https://', '') }}</a>
-        {% if request.user.is_superuser %}Score: <span title="explanation: {{ doc.explanation }}">{{ doc.score }}</span>{% endif%}
+        {% if request.user.is_superuser %}Score: {{ doc.score }}{% endif%}
```
Comment (robhudson) [inline kuma/search/templates/search/results.html]: An extra space in the `endif` tag please.

**Comment by robhudson** [comment]
Since it seems like tests didn't catch this bug, it might be worthwhile to make sure we have a test hitting the search.json endpoint and a basic check that it contains what we expect.

r+wc

File: kuma/search/templates/search/results.html (hunk 1 above)
Comment (jezdez) [inline kuma/search/templates/search/results.html]: Good catch!

**Comment by jezdez** [comment]
@robhudson r?

**Comment by robhudson** [comment]
awesome, thanks. r+

**Comment by jezdez** [comment]
just flake8 silliness

**Pull request merged**